src/platform/site-wide/tests/alerts/MobileAppCallout.unit.spec.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { expect } from 'chai';
+import { render } from '@testing-library/react';
+
+import MobileAppCallout from '@department-of-veterans-affairs/platform-site-wide/MobileAppCallout';
+
+describe('MobileAppCallout', () => {
+  describe('when it exists', () => {
+    it('should render mobile app callout correctly with App Store and Google Play buttons', () => {
+      const view = render(<MobileAppCallout />);
+      expect(view.queryByText(/your mobile device/i)).to.exist;
```

Comment:
Because the header is dynamic that text doesn't exist any more.

Reply:
You could improve the test to pass different text in as a prop when you render and test that that text exists...